cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,97 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignUpPage from '../support/pages/signUp.pageObject';
+import ArticlePage from '../support/pages/article.pageObject';
+import faker from 'faker';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('User', () => {
-  before(() => {
+  let user;
+  let name;
+  let email;
+  let password;
+  let title;
+  let description;
+  let content;
+  let tags;
+  const signUpPage = new SignUpPage();
+
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      name = faker.name.firstName();
+      email = faker.internet.email();
+      password = faker.internet.password();
+
+      cy.register(user.email, user.username, user.password);
+      cy.visit('/#/register');
+
+      signUpPage.fillName(name);
+      signUpPage.fillEmail(email);
+      signUpPage.fillPassword(password);
+
+      signUpPage.submitSignUpForm();
+      cy.contains('OK').click();
+      homePage.clickNewArticle();
 
+      title = faker.lorem.words(2);
+      description = faker.lorem.words(2);
+      content = faker.lorem.words(2);
+      tags = 'tag1';
+
+      articlePage.fillTitle(title);
+      articlePage.fillDescription(description);
+      articlePage.fillContent(content);
+      articlePage.fillTags(tags);
+      articlePage.submitArticle();
+
+      homePage.clickSettings();
+      cy.scrollTo('bottom');
+      settingsPage.logOutSettings();
+
+      cy.visit('/#/register');
+      signUpPage.fillName('user2');
+      signUpPage.fillEmail('user2@mail.com');
+      signUpPage.fillPassword('User123!');
+      signUpPage.submitSignUpForm();
+
+      cy.contains('OK').click();
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
+  it('should be able to follow the another user', () => {
```

Comment:
The 'follow' and 'unfollow' tests have repeated code (lines 73-78 and 86-92 are duplicates). Consider creating a helper function to reduce repetition.

Reply:
I changed my code and left in tests 2 and 3 rows only